fix(PRO-5714): pro-use workflowGuid to fix duplicated events when comitting (#53)
apostrophecms/apostrophe-events: CHANGELOG.md

> ## UNRELEASED

Author: ```suggestion
## UNRELEASED

⚠️ ☝️ MAJOR version should be incremented to `3.0.0`. Please remove this warning when doing so during the release.
```

> ### UNRELEASED

Author: ```suggestion
## UNRELEASED
```

for the parsing to work, @boutell could you confirm that we should not use 3 `#` anymore?

Reviewer: Use two

Reviewer 2: Changed it to two

> * When repeating an event, we were basing our knowledge of whether or not this event already had duplicates, on its parent's `id` but its `id` would change between versions. This resulted in infinite duplicates when committing an event with the `repeat` property. This has been changed to check that its parent's `workflowGuid` does not change across document versions.

Author: ```suggestion
* When repeating an event, we were basing our knowledge of whether or not this event already had duplicates, on its parent's `id` but its `id` would change between versions. This resulted in infinite duplicates when committing an event with the `repeat` property. This has been changed to check that its parent's `workflowGuid` does not change across document versions.

**BREAKING CHANGE:** projects should now use `workflowGuid` and `workflowLocale` fields to get the parent or children events and not `_id` anymore.```

> * Although it caused no apparent issue, the pieces that were dupplicated in a dupplicated event had its parents `hasClones` property set to true (since it was dupplicated and unchanged after that). To make more sense in the pieces, the dupplicates now have the `nowClones` property set to false upon cloning.

Author: ```suggestion
* Although it caused no apparent issue, the pieces that were duplicated in a duplicated event had its parents `hasClones` property set to true (since it was duplicated and unchanged after that). To make more sense in the pieces, the duplicates now have the `nowClones` property set to false upon cloning.
```

> * Although it caused no apparent issue, the pieces that were dupplicated in a dupplicated event had its parents `hasClones` property set to true (since it was dupplicated and unchanged after that). To make more sense in the pieces, the dupplicates now have the `nowClones` property set to false upon cloning.

Author: ```suggestion
* Although it caused no apparent issue, the pieces that were dupplicated from an event of type "repeat" had its parents `hasClones` property set to true (since it was dupplicated and unchanged after that). To make more sense in the pieces, the dupplicates now have the `nowClones` property set to false upon cloning.
```

> * Although it caused no apparent issue, the pieces that were duplicated from an event of type `repeat` had its parent's `hasClones` property set to true (since it was duplicated and unchanged after that). To make more sense in the pieces, the duplicates now have the `nowClones` property set to false upon cloning.

Author: ```suggestion
* Although it caused no apparent issue, the pieces that were duplicated from an event of type `repeat` had its parent's `hasClones` property set to true (since it was duplicated and unchanged after that). To make more sense in the pieces, the duplicates now have the `hasClones` property set to false upon cloning.
```
